src/pudl/etl.py

```diff
@@ -370,6 +350,35 @@ def _etl_glue(glue_settings: GlueSettings) -> dict[str, pd.DataFrame]:
     return glue_dfs
 
 
+def _read_static_encoded_tables(
+    etl_group_name: Literal["static_eia", "static_ferc1"]
```

Comment:
Why not just name this parameter `etl_group` since that's the element it's going to look at.